pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -173,6 +188,36 @@ func (i *CommandInitOption) Validate(parentCommand string) error {
 	return nil
 }
 
+func (i *CommandInitOption) validateExternalEtcd(_ string) error {
+	if i.ExternalEtcdCACertPath == "" {
+		return fmt.Errorf("etcd ca cert path should be specified")
+	}
+	if i.ExternalEtcdClientCertPath == "" {
+		return fmt.Errorf("etcd client cert path should be specified")
+	}
+	if i.ExternalEtcdClientKeyPath == "" {
+		return fmt.Errorf("etcd client cert private key path should be specified")
+	}
```

Comment:
I think this is optional and not required.

Reply:
OK, I will make these optional.